src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,14 @@
+package duke.commands;
```

Comment:
I think its a good call to package the different types of commands under this class.